packages/sync-repo-settings/src/sync-repo-settings.ts

```diff
@@ -154,30 +167,43 @@ handler.updateMasterBranchProtection = async function updateMasterBranchProtecti
   }
 };
 
-/**
- * Ensure the correct teams are added to the repository
- * @param repos List of repos to iterate.
- */
-handler.updateRepoTeams = async function updateRepoTeams(
-  repo: Repo,
-  context: Context
-) {
-  console.log(`Update team access for ${repo.repo}`);
-  const [owner, name] = repo.repo.split('/');
-  const teamsToAdd = [
+handler.defaultLanguageTeams = function (language: string): TeamPermission[] {
+  return [
     {
       slug: 'yoshi-admins',
       permission: 'admin',
     },
     {
-      slug: `yoshi-${repo.language}-admins`,
+      slug: `yoshi-${language}-admins`,
       permission: 'admin',
     },
     {
-      slug: `yoshi-${repo.language}`,
+      slug: `yoshi-${language}`,
       permission: 'push',
     },
   ];
+};
+
+handler.getRepoTeams = function (repo: Repo): TeamPermission[] {
+  const language = repo.language;
+  let teams = handler.defaultLanguageTeams(language);
+  if (language in languageTeams) {
+    teams = teams.concat(languageTeams[language]);
```

Comment:
You could keep `teams` a `const`, and do `teams.push(...languageTeams[language])` as well.  (non-blocking feedback)

Reply:
Done